freetextresponse/freetextresponse.py

```diff
@@ -318,17 +334,16 @@ def _get_word_count_message(self, ignore_attempts=False):
             )
         return result
 
-    def _get_indicator_class(self):
+    def _get_indicator_class(self, student_answer_type=None):
```

Comment:
student_answer_type=None is an important part of the code.  It's how we get the 'unanswered' indicator.  There are three strings, 'unanswered', 'correct', and 'incorrect'